backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt

```diff
@@ -160,8 +160,8 @@ fun BpkNudger(
                         text = subtitle,
                         style = BpkTheme.typography.bodyDefault,
                         color = BpkTheme.colors.textSecondary,
-                        maxLines = 1,
-                        overflow = TextOverflow.Ellipsis,
+                        maxLines = Int.MAX_VALUE,
+                        overflow = TextOverflow.Clip,
```

Comment:
nit: both lines can be removed as we're just using the default values

Reply:
Oh, I just see.

Reply:
Yeah, It upgraded.